central/vulnmgmt/vulnerabilityrequest/manager/requestmgr/manager_impl.go

```diff
@@ -577,21 +581,27 @@ func (m *managerImpl) getFixableDeferrals() ([]*storage.VulnerabilityRequest, er
 	}
 	var fixableReqs []*storage.VulnerabilityRequest
 	for _, res := range results {
-		for _, cve := range res.GetCves().GetCves() {
+		for _, CVE := range res.GetCves().GetCves() {
 			qb := search.NewQueryBuilder().AddBools(search.Fixable, true)
 			// TODO: Determine if it's worth checking cvePkg.ContainsComponentBasedCVE(cve.GetTypes()) before doing this. It would involve going to the data store to read CVE data
 			// This is only necessary if somehow there ended up being a deferral on a cluster CVE. Or if it goes from an image cve to node cve
 			// TODO: Test what happens if it's a cluster cve
 			ctx := allAccessCtx
-			qb.AddExactMatches(search.CVE, cve)
+			qb.AddExactMatches(search.CVE, CVE)
 
 			fixableQuery, err := utils.GetAffectedImagesQuery(res, qb.ProtoQuery())
 			if err != nil {
 				return nil, err
 			}
-			count, err := m.imageCVEs.Count(ctx, fixableQuery)
+			var count int
+			if features.FlattenCVEData.Enabled() {
+				count, err = m.ImageCVEView.Count(ctx, fixableQuery)
+
+			} else {
+				count, err = m.imageCVEs.Count(ctx, fixableQuery)
+			}
```

Comment:
Now that I look at this again, I wonder if we can just use `m.ImageCVEView.Count(ctx, fixableQuery)` for the old and the new.  What do you think?